src/component/chatting/chattingContent.tsx

```diff
@@ -0,0 +1,49 @@
+import styled from 'styled-components';
+import Talk from './talk';
+import { ChattingRoom } from './../../interface/interface';
+import { useRecoilValue } from 'recoil';
+import { chatList } from './../../atom/atom';
+import { useRef, useEffect } from 'react';
+
+const ChattingContent = ({ chattingRoomId }: ChattingRoom) => {
+  const chattingList = useRecoilValue(chatList);
+
+  const scrollRef = useRef<HTMLDivElement | null>(null);
+
+  const scrollBottom = () => {
+    if (scrollRef.current) {
+      scrollRef.current.scrollTop = scrollRef.current.scrollHeight;
+    }
+  };
```

Comment:
저는 이 부분 코드가 상당히 불편해서 (null을 굳이 주어야 하는가... null값에 대해 항상 생각해야하는가? 에 대해 불편했어요)
그래서 아래와 같이 작성하는 방법을 선택했습니다. 참고해보아도 좋을 것 같아요~!
```suggestion
  const scrollRef = useRef<HTMLDivElement>(null);

  const scrollBottom = () => {
    scrollRef?.current?.scrollTop = scrollRef?.current?.scrollHeight;
  };
```
~지금 제대로 생각이 안나서 ? 투성이지만~ optional chanining (?.)을 사용하면 코드 길이를 좀 줄이면서 null값에 대해 예외처리도 한번에 되어서 좋은 것 같아요.